src/Entities/Entity.h

```diff
@@ -530,6 +535,11 @@ class cEntity
 	eEntityType m_EntityType;
 	
 	cWorld * m_World;
+
+	/// State variables for ScheduleMoveToWorld.
+	bool m_IsWorldChangeScheduled = false;
```

Comment:
Shouldn't this be initialized in the construcor? That is where we usualy initialize member variables right?